pages/clubs/Running.jsx

```diff
@@ -43,40 +43,40 @@ var Running = React.createClass({
   componentDidMount: function() {
     this.fetchJSON("https://mozilla.github.io/learning-networks/clubs/clubs-resources.json",this.setData);
   },
-  getLinks : function(){
-    var categories = this.state.guides.map(function(category){
-      var guideLinks = category.guides.map(function(guide){
-
-        if(guide.translations && guide.translations.length > 0){
-          var translationLinks = guide.translations.map(function(translation){
-            return (
-              <li>
```

Comment:
each map needs a key, so both functions need a `key` in their return element (we had three spots)